docs/rackhd/graph_progress.rst

```diff
@@ -0,0 +1,146 @@
+Workflows Progress
+---------------------------
+
+RackHD workflow progress feature provide message notification mechanism to indicate status of an active workflow or task. User can get to know what have been done and what is left for an active workflow or task with progress message. It is helpful when user is running time-consuming workflows like OS installation or secure erase.
+
+Progress Message Publishment Criteria
```

Comment:
I didn't catch up the meaning at first glance - a little hard to understand:-). How about changing it to "What kind of progress message is supported"?